verified-inference/host/api/repository/nitro_cli_repository.py

```diff
@@ -4,7 +4,7 @@
 
 
 TEE_CPU_COUNT = 2
-TEE_MEMORY_IN_MB = 8192
+TEE_MEMORY_IN_MB = 16384
```

Comment:
Why do we need to increase it to 16GB?

Reply:
with the smolagents, the minimal RAM requirement is around 14GB